<ui version="4.0" >
 <class>PopupWinClass</class>
 <widget class="QWidget" name="PopupWinClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>179</width>
    <height>81</height>
   </rect>
  </property>
  <property name="contextMenuPolicy" >
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle" >
   <string>PopupWin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="2" >
      <widget class="QLabel" name="lblTitle" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags" >
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QLabel" name="lblText" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>130</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>200</width>
         <height>400</height>
        </size>
       </property>
       <property name="text" >
        <string>TextLabel</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
       <property name="openExternalLinks" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="lblSidebar" >
       <property name="minimumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap>://Resources/Document.png</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
       <property name="alignment" >
        <set>Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="res.qrc" />
 </resources>
 <connections/>
</ui>
